src/utils.ts

```diff
@@ -1,7 +1,70 @@
 import { constants } from './prelude';
-import { ethers } from 'hardhat';
+import hre, { ethers } from 'hardhat';
```

Comment:
Wow, did not know that such syntax is valid. We should apply it to other repos as well.